code_samples/ai_actions/src/AI/Handler/LLaVaTextToTextActionHandler.php

```diff
@@ -0,0 +1,78 @@
+<?php declare(strict_types=1);
+
+namespace App\AI\Handler;
+
+use Ibexa\Contracts\ConnectorAi\Action\ActionHandlerInterface;
+use Ibexa\Contracts\ConnectorAi\Action\DataType\Text;
+use Ibexa\Contracts\ConnectorAi\Action\Response\TextResponse;
+use Ibexa\Contracts\ConnectorAi\Action\TextToText\Action as TextToTextAction;
+use Ibexa\Contracts\ConnectorAi\ActionInterface;
+use Ibexa\Contracts\ConnectorAi\ActionResponseInterface;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+
+final class LLaVaTextToTextActionHandler implements ActionHandlerInterface
+{
+    private HttpClientInterface $client;
+
+    private string $host;
+
+    public const string IDENTIFIER = 'LLaVATextToText';
```

Comment:
```suggestion
    public const IDENTIFIER = 'LLaVATextToText';
```

('won't work otherwise in PPH 7.4)

Reply:
Great catch, thank you! I will adjust the CI setup here to run PHPStan on PHP 7.4 as well